kuma/wiki/tests/test_forms.py

```diff
@@ -179,51 +181,51 @@ def test_case_sensitive_tags(self, mock_requests):
         self.assertTrue(rev_form.is_valid())
         self.assertEqual(rev_form.cleaned_data['tags'], '"JavaScript"')
 
-    @pytest.mark.spam
-    @requests_mock.mock()
-    def test_akismet_enabled(self, mock_requests):
+    def _setup_akismet_post(
+            self, mock_requests, extra_data=None, revision=None, is_spam='false'):
+        """Setup a RevisionForm for content checking with Akismet.
```

Comment:
I'm not a fan of triple quotes without a linebreak since `"""Something something"""` is less readable than

```
"""
Something something
"""
```